### PR TITLE
[DPMMA-2341] Fixed missing preference center

### LINKED ISSUE #12828: "Preference Center Page" input is missing in the email form
### Mautic Version

5.0.x series

### PHP version

8.0.20

### What browsers are you seeing the problem on?

Not relevant

### What happened?

"Preference center page" input is missing in the email form (s/emails/edit/{id} or s/emails/new)

Mautic v4
email_v4

Mautic v5
email_v5

### How can we reproduce this issue?

Step 1: Create or edit email
Step 2: Try setting up a custom preferences page

### Relevant log output

_No response_

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by annamunk**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✅]
| New feature/enhancement? (use the a.x branch)      | [❌]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [❌] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | #12828

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Log in to Mautic using an account that has page:preference_center:viewown and page:preference_center:viewother permissions.
3. Create or edit email.
4. Find the "Preference Center Page" input.

### CODE REVIEW

File: app/bundles/EmailBundle/Resources/views/Email/form.html.twig [hunk 1]
```diff
@@ -181,7 +181,7 @@
                 {% endif %}
 
                 {{ form_row(form.unsubscribeForm) }}
-                {% if permissions['page:preference_center:viewown'] is not empty and permissions['page:preference_center:viewother'] is empty %}
+                {% if permissions['page:preference_center:viewown'] is not empty and permissions['page:preference_center:viewother'] is not empty %}
```
Comment (mollux) [inline app/bundles/EmailBundle/Resources/views/Email/form.html.twig]:
```suggestion
                {% if permissions['page:preference_center:viewown'] is not empty or permissions['page:preference_center:viewother'] is not empty %}
```

**Review by mollux — CHANGES_REQUESTED** [review]
@annamunk 

while the code has the correct outcome, it differs from the approach in M4, see https://github.com/mautic/mautic/blob/4.4.10/app/bundles/EmailBundle/Views/Email/form.html.php#L224...L227

in M4 one of the 2 permissions was enough, in M5 both are needed

**Comment by annamunk** [comment]
@mollux thanks for catching it! I commited the change.

**Review by kuzmany — APPROVED** [review]
I noticed same during the build https://github.com/mautic/mautic/pull/12860
This fixed it 👍

**Review by volha-pivavarchyk — APPROVED** [review]
Works well.

Thanks. I faced the same problem some time ago.

**Pull request merged**